### PR TITLE
moving scientific ownership to sam lee instead of laura

### PR DESCRIPTION
**Pull request by nikellepetrillo**

### CODE REVIEW

**Review by khajoue2 — APPROVED** [review]

File: .pullapprove.yml [hunk 1]
```diff
@@ -160,7 +164,7 @@ groups:
       request: 2
     reviewers:
       users:
-        - ldgauthier # Laura Gauthier
+        - samuelklee # Samuel Lee
```
Comment (samuelklee) [inline .pullapprove.yml:167]: I'm happy to help review any changes to these pipelines, but strictly speaking, I think Laura only handed off responsibilities for the joint-genotyping pipelines to me. Perhaps @kachulis can clarify who the primary owner for the single-sample pipelines will be going forward?

Comment (samuelklee) [reply .pullapprove.yml:167]: (Or if it turns out that I do own these now, it would be good to sort that out!)

Comment (samuelklee) [reply .pullapprove.yml:167]: OK---chatted with Chris and I understand the intended split now. He'll own everything up through alignment/duplicate marking.

**Review by jessicaway — APPROVED** [review]

File: .pullapprove.yml (hunk 1 above)
Comment (nikellepetrillo) [reply .pullapprove.yml:167]: thanks sam! that sounds good 👍

**Pull request merged**